presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -819,3 +819,26 @@ row. The returned value is ``NULL`` if the subquery produces no rows::
     WHERE regionkey = (SELECT max(regionkey) FROM region)
 
 .. note:: Currently only single column can be returned from the scalar subquery.
+
+LATERAL
+^^^^^^^
+Subqueries appearing in the FROM clause can be preceded by the key word LATERAL.
+This allows them to reference columns provided by preceding FROM items.
+
+A LATERAL item can appear at top level in the FROM list, or within a JOIN tree.
+In the latter case it can also refer to any items that are on the left-hand side
+of a JOIN that it is on the right-hand side of.
+
+When a FROM item contains LATERAL cross-references, evaluation proceeds as follows:
+for each row of the FROM item providing the cross-referenced column(s),
+the LATERAL item is evaluated using that row or row set's values of the columns.
+The resulting row(s) are joined as usual with the rows they were computed from.
+This is repeated for each row or set of rows from the column source table(s).
+
+LATERAL is primarily useful when the cross-referenced column is necessary for
+computing the row(s) to be joined. Please see the below example:
```

Comment:
Use a double `::` so that it is a code block in the default language (SQL). Also, we can simplify to
```
``LATERAL`` is primarily useful when the cross-referenced column is necessary for
computing the row(s) to be joined::